superchain/configs/sepolia/ink.toml

```
@@ -0,0 +1,58 @@
+name = "Ink Sepolia"
+chain_id = 763373
+public_rpc = "https://rpc-gel-sepolia.inkonchain.com"
+sequencer_rpc = "https://rpc-gel-sepolia.inkonchain.com"
+explorer = "https://explorer-sepolia.inkonchain.com"
+superchain_level = 0
+standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
+batch_inbox_addr = "0x004De1914F4f17aC234f5A5Bc8a4072a231d44BF"
+canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
```

```suggestion
superchain_time = 0
```
They also activated Holocene at the Superchain time and probably want to continue following superchain activations going forward.

This is also related to us trying to add chains with superchain_time by default unless there are objections. cc @sbvegan

---

Wouldn't removing the `<hardfork>_time = 0` so that it inherits the default timestamps change the config? Or does it not matter since the default timestamp values are < the genesis timestamp so they all get activated at genesis either way?

---

it doesn't matter, the resulting config is the same, only that now also Holocene is pulled in.